tests/test_management.py

```diff
@@ -257,8 +351,7 @@ def test_transmit_course_metadata_task_no_channel(self):
     @responses.activate
     def test_transmit_course_metadata_task_no_catalog(self):
         """
-        Test the data transmission task with enterprise customer which have no
-        course catalog.
+        Test the data transmission task with enterprise customer which have no course catalog.
```

Comment:
@UmanShahzad Typo: "which have" should be "that has" (or "who has")

Reply:
:+1: